src/pages/workspace/WorkspacesListRow.tsx

```diff
@@ -147,16 +152,24 @@ function WorkspacesListRow({
                 >
                     {title}
                 </Text>
-                {isNarrow && (
-                    <>
-                        <BrickRoadIndicatorIcon brickRoadIndicator={brickRoadIndicator} />
-                        <ThreeDotsMenu
-                            menuItems={menuItems}
-                            anchorPosition={{horizontal: 0, vertical: 0}}
-                            disabled={shouldDisableThreeDotsMenu}
+                {isNarrow &&
+                    (!isJoinRequestPending ? (
```

Comment:
The styling is broken at screen **width** 800 to 900px on web.

<img width="416" alt="image" src="https://github.com/Expensify/App/assets/29673073/aaaa0644-043f-419d-95eb-ccbd068b7506">

Reply:
I think we do not have space here - based on the column structure of WorkspacesListRow, do we know how it should be? @luacmartins

Reply:
Hmm I feel like it might be beneficial to trigger the breakpoint a bit earlier. Even if it's just an hour glass icon, the rest of the columns are barely legible. 

What does this look like on mobile btw?

Reply:
@shawnborton 
![Simulator Screenshot - iPhone 15 Pro - 2024-03-07 at 20 36 28](https://github.com/Expensify/App/assets/28352309/c9c6eea7-cb19-4e4f-83f6-1d5f467c8452)

Reply:
Got it, for mobile, I would expect there to be equal padding on the left/right side of the content. So the right side of the badge should match the padding on the left side of the workspace icon:
![CleanShot 2024-03-07 at 13 38 14@2x](https://github.com/Expensify/App/assets/2319350/c6a1b466-16c8-4f79-8a3c-ab5522b719dc)

Reply:
@shawnborton we can have break point on medium size <1024px
<img width="1298" alt="Screenshot 2024-03-07 at 20 39 32" src="https://github.com/Expensify/App/assets/28352309/9fc5fa03-ae1a-49d2-9921-ee3d304b50dd">

Reply:
Yeah I guess what I am saying is, we should flip over to these styles:
![CleanShot 2024-03-07 at 13 44 22@2x](https://github.com/Expensify/App/assets/2319350/09a9f292-f36e-453e-a557-6a72d6cef797)

A bit sooner, perhaps at the medium size like you suggest. Thoughts on something like that?

Reply:
Yup we have medium breakpoint which finished at 1025px - and we can apply design from mobile version from 1024px @shawnborton

Reply:
Cool, let's see how it looks first and decide from there?

Reply:
@shawnborton you can check my above message - i already attached 2 images with 1025px and less 1025

Reply:
Ah I missed the second image. I think that looks good, curious if the other designers agree. 

Can you address my padding comment too please? Thanks!

Reply:
Sure - already did it
![Simulator Screenshot - iPhone 15 Pro - 2024-03-07 at 21 05 13](https://github.com/Expensify/App/assets/28352309/6784a57b-2969-42fa-be04-f9f95f419b31)
<img width="907" alt="Screenshot 2024-03-07 at 21 05 07" src="https://github.com/Expensify/App/assets/28352309/cb903e59-2823-4970-b24f-922bbb10351a">

Reply:
resolved. 

<img width="306" alt="image" src="https://github.com/Expensify/App/assets/29673073/2065cc4a-8c2e-4e56-a5a2-9eda507a5610">

Reply:
Updated padding looks great and I like swapping to the "mobile" styles at the medium breakpoint!

Reply:
I agree!